cfgov/unprocessed/css/molecules/info-unit.less

```diff
@@ -0,0 +1,161 @@
+@m-info-unit_img__sm: 130px;
+@m-info-unit_img__lg: 150px;
+
+.m-info-unit {
+    &__inline {
+        .u-clearfix();
+    }
+
+    &__inline &_image {
+        .respond-to-min( @bp-sm-min, {
+            float: left;
+            margin-bottom: 0;
+        } );
+    }
+
+    &__home &_image {
+        .respond-to-range( @bp-sm-min, @bp-med-max, {
+            float: left;
+            margin-bottom: 0;
+        } );
+    }
+
+    &__inline &_content {
+        .respond-to-min( @bp-sm-min, {
+            margin-left: @m-info-unit_img__sm + @grid_gutter-width;
+        } );
+
+        .respond-to-min( @bp-med-min, {
+            margin-left: @m-info-unit_img__lg + @grid_gutter-width;
+        } );
+    }
+
+     &__home &_content {
+        .respond-to-range( @bp-sm-min, @bp-sm-max, {
+            margin-left: @m-info-unit_img__sm + @grid_gutter-width;
+        } );
+
+        .respond-to-range( @bp-med-min, @bp-med-max, {
+            margin-left: @m-info-unit_img__lg + @grid_gutter-width;
+        } );
+    }
+
+    &_image {
+        margin-right: auto;
+        margin-bottom: 15px;
+        margin-left: auto;
+        display: block;
+        max-height: @m-info-unit_img__sm;
+        background-size: cover;
+
+        .respond-to-min(@bp-med-min, {
+            max-height: @m-info-unit_img__lg;
+        });
+
+        &__square {
+            width: @m-info-unit_img__sm;
+            height: @m-info-unit_img__sm;
+
+            .respond-to-min( @bp-med-min, {
+                width: @m-info-unit_img__lg;
+                height: @m-info-unit_img__lg;
+            } );
+        }
+    }
+
+    &_btn {
+        .respond-to-max( @bp-xs-max, {
+            width: 100%;
+            text-align: center;
+        } );
+    }
+}
+
+.content_main__home {
+    padding-top: 30px;
+
+    .respond-to-max( @bp-xs-max, {
+        padding-bottom: 60px;
+    } );
+}
+
+.content-l_col__home {
+    .respond-to-max( @bp-med-max, {
+        + .content-l_col__home {
+            margin-top: 30px;
+        }
+    } );
+
+    .respond-to-min( @bp-sm-min, {
+        .grid_column( 1, 1 );
+    } );
+
+    .respond-to-min( @bp-lg-min, {
+        .grid_column( 4 );
+    } );
+}
+
+.o-well__home {
+    .respond-to-min( @bp-xl-min, {
+        max-width: 1230px - ( 45px * 2 ) !important;
+        margin: 0 auto;
+        padding-right: 45px !important;
+        padding-left: 45px !important;
```

Comment:
Is it really an `o-well` if it's being overridden with `!important`? Perhaps it's a separate organism.